### DISCUSSION TITLE
Sponsorship in active sponsors but no email/activity feed/stripe transaction

### DISCUSSION THREAD

**Discussion by rianadon** [community/community#68012] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I ran into a weird occurrence where someone sponsored me, but I didn't receive the usual new-sponsorship email. The sponsorship also additionally:
1. Is missing from the Sponsors activity feed in the dashboard
2. Is missing from the Stripe dashboard
3. Is missing when I go to my user profile page and look at the list of sponsor profile pictures

This only place the sponsorship shows up is the list of active sponsors in my sponsorship dashboard.

![image](https://github.com/community/community/assets/18223213/452a1c1c-9e82-40f6-b7c3-50c83201b504)

I checked with the person who sponsored me and from their end, GitHub reports that they're a sponsor. So it looks like whatever kicks off the email/stripe transaction didn't happen?

<img width="531" alt="image" src="https://github.com/community/community/assets/18223213/78f7b94d-8b56-4764-9633-d21d91824619">

I'm not sure if it's the same issue, but this sounds similar to https://github.com/orgs/community/discussions/53584.

**Comment by underwoo16** [comment]
Hey @rianadon, I’m one of the engineers on GitHub Sponsors. Thanks for sharing this bug! Our intention is to make all sponsorship transactions smooth and easy, but unfortunately, that didn’t happen in this case. 

We were seeing intermittent issues processing payments when this sponsorship was made, and we had to cancel the unpaid sponsorship. We’ve reached out to the user who created the sponsorship to let them know about the error and invited them to try again.

We appreciate your patience! Do you mind leaving a reply once the sponsorship is (hopefully) successfully created? It would be great to know that you’re good! If you run into another issue, I’d be happy to further assist you.

**Reply by rianadon** [reply]
Thanks @underwoo16 for taking a look at this and for doing the extra work of reaching out! The user tried again and it looks like the sponsorship processed correctly this time. It's now showing up in all the places I'd expect it to. Thank you again for the help!

**Reply by rianadon** [reply]
~~Hi @underwoo16. Sponsors has been working super smoothly since your reply, but today it looks like I've run into the same problem. I have a sponsorship from today (Nov 27) that shows up in active sponsors but is missing from the activity feed and stripe. Thanks for looking into this again!~~

Nevermind! It just took a few hours to process this time.

**Comment by ad1992** [comment]
Hi, I am seeing a similar weird sponsorship issue today, my sponsorship is listed on "Active Sponsors" but missing from the below areas 👇🏻 

1. I didn't receive any email notification about the new sponsorship
2. Is missing from the Sponsors activity feed in the dashboard
3. Is missing from the Stripe dashboard
4. It Is missing when I go to my user profile page and look at the list of sponsor profile pictures

Any help would be appreciated 🙏

**Reply by ad1992** [reply]
EDIT: ok I received it now, nvm looks like there was a delay

**Comment by preeti13456** [comment]
How much time it takes to get the sponsor amount into your account successfully delivered ?
